src/plugins/intel_gpu/tests/functional/single_layer_tests/dynamic/mvn.cpp

```diff
@@ -283,29 +247,5 @@ const auto Mvn1D = ::testing::Combine(
 
```

Comment:
What about test case above? I think all test cases which use `empty_reduction_axes` and `empty_reduction_axes` itself should be removed.

Reply:
Updated